web/user.go

```diff
@@ -56,6 +56,19 @@ func (u *UserInfo) GetBool(name string) bool {
 	return b
 }
 
+// GetLastLogin gets the user last login time.
+func (u *UserInfo) GetLastLogin() string {
+	if u.Properties == nil {
+		return ""
+	}
+
+	ll, ok := u.Properties["updated_at"].(string)
```

Comment:
can we be verbose and name the variable `lastLogin`?
is there a reason why there's a discrepancy between the name `updated_at` and `last_login`?

Reply:
The reason for the discrepancy between the names 'updated_at' and 'last_login' in the code is because 'last_login' wasn't directly accessible on the auth0 user object. In order to retrieve the timestamp indicating when the user's profile was last updated or modified, I had to use the 'updated_at' field instead. Changes to 'last_login' are considered updates, so 'updated_at' will match 'last_login'. Therefore, I decided to use 'updated_at' to fulfill the purpose of retrieving the user's last login time.

Reply:
will they ever be out of sync? for example, will `updated_at` potentially be more recent than `last_login`?

Reply:
In most cases, 'updated_at' and 'last_login' will match because changes to 'last_login' are considered updates and will update the 'updated_at' timestamp as well. However, there could be scenarios where 'updated_at' might be more recent than 'last_login'. One such scenario could be when the user's profile is modified without triggering a login event. For example, if an administrator updates the user's profile information without the user logging in, 'updated_at' would be updated while 'last_login' remains unchanged.

Reply:
oh okay that does seem unusual. sounds good then, maybe just leave a comment here or link to info when `updated_at` is updated?

Reply:
Good call, thanks - added in a comment explaining 'updated_at.'